test/NuGet.Core.Tests/NuGet.Commands.Test/PackCommandRunnerTests.cs

```diff
@@ -168,6 +227,47 @@ public void Dispose()
             {
                 _testDirectory.Dispose();
             }
+
+
+            internal static void ValidNuspec(string nuspecPath, string packageId, string packageVersion, string pattern)
+            {
+                File.WriteAllText(nuspecPath,
+                    $@"<?xml version=""1.0""?>
+                    <package>
+                        <metadata>
+                            <id>{packageId}</id>
+                            <version>{packageVersion}</version>
+                            <title>title</title>
+                            <description>description</description>
+                            <authors>author</authors>
+                            <requireLicenseAcceptance>false</requireLicenseAcceptance>
+                            <dependencies />
+                        </metadata>
+                        <files>
+                            <file src=""{pattern}"" target="""" />
+                        </files>   
+                    </package>");
+            }
+
+            internal static void InvaidNuspecNoVersion(string nuspecPath, string packageId, string pattern)
```

Comment:
is this method just a special case of `ValidNuspec` where `string.Empty` is passed to the `packageVersion` argument?